test/e2e/autoscale_test.go

```diff
@@ -107,13 +107,18 @@ func (suite *AutoscaleTestSuite) TestAutoScaleIngester() {
 	waitUntilScales(jaegerInstanceName, "ingester")
 }
 
-func createAndWaitFor(jaegerInstance *v1.Jaeger, jaegerInstanceName string) {
+func createAndWaitFor(jaegerInstance *v1.Jaeger, jaegerInstanceName string, waitForIngester bool) {
 	err := fw.Client.Create(context.TODO(), jaegerInstance, &framework.CleanupOptions{TestContext: ctx, Timeout: timeout, RetryInterval: retryInterval})
 	require.NoError(t, err, "Error deploying example Jaeger")
 
 	err = e2eutil.WaitForDeployment(t, fw.KubeClient, namespace, jaegerInstanceName+"-collector", 1, retryInterval, timeout)
 	require.NoError(t, err, "Error waiting for collector deployment")
 
+	if waitForIngester {
```

Comment:
Is this generally useful, or only specific to the otel images? If later, maybe we should skip these tests for now if the `specifyOtelConfig` flag is true?